Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -75,6 +77,7 @@ open class ComposerVC: _ViewController,
             threadMessage: ChatMessage?,
             attachments: [AnyAttachmentPayload],
             mentionedUsers: Set<ChatUser>,
+            extraData: [String: RawJSON] = [:],
             command: Command?
```

Comment:
please add the extra data as the last argument, especially since it is optional. If someone is using the ´command` it will cause a breaking change, since the order of the arguments will change

Reply:
Since the default argument is used, this will not break the API. I can change the order though ;)

Reply:
Ah yes, you are right 👍 Anyways, better to add the extra data to the last item 👍

Reply:
@nuno-vieira Done ;)